service/src/main/java/uk/gov/hmcts/reform/fpl/service/payment/PaymentService.java

```diff
@@ -79,13 +80,17 @@ public void makePaymentForC2(Long caseId, CaseData caseData) {
         CreditAccountPaymentRequest paymentRequest = getCreditAccountPaymentRequest(caseId,
             c2DocumentBundle.getPbaNumber(),
             c2DocumentBundle.getClientCode(),
-            c2DocumentBundle.getFileReference(),
+            defaultCustomerReference(c2DocumentBundle.getCustomerReference()),
```

Comment:
Could we specify that's only default when blank? It suggest we always use default?